lib/protobuf/dsl.ex

```diff
@@ -123,14 +129,17 @@ defmodule Protobuf.DSL do
           def value(v) when is_integer(v), do: v
         end
       ] ++
-      Enum.map(props, fn {_, %{fnum: fnum, name_atom: name_atom}} ->
+      Enum.map(atom_mapping, fn {name_atom, fnum} ->
         quote do
           def key(unquote(fnum)), do: unquote(name_atom)
         end
       end) ++
       [
         quote do
-          def mapping(), do: unquote(Macro.escape(mapping))
+          def mapping(), do: unquote(Macro.escape(atom_mapping))
+        end,
+        quote do
+          def reverse_mapping(), do: unquote(Macro.escape(reverse_mapping))
```

Comment:
Should probably name it `__reverse_mapping__/0` instead, since this is kind of private API.